api/index.js

```diff
@@ -5,7 +5,7 @@ import axios from 'axios'
 import { NetworkError } from './errors'
 
 const CORE_API = 'https://api.core.ac.uk/internal'
-const CORE_API_DEV = 'https://api.dev.core.ac.uk/internal'
```

Comment:
Not a good idea. A cookie will not be sent.

Reply:
Yeah, I know 😔